Base/Enums/BacnetEventTypes.cs

```diff
@@ -14,6 +14,16 @@ public enum BacnetEventTypes
     EVENT_EXTENDED = 9,
     EVENT_BUFFER_READY = 10,
     EVENT_UNSIGNED_RANGE = 11,
+    EVENT_ACCESS_EVENT = 13,
```

Comment:
Can you add comment, that 12 is reserved for future addenda?

Reply:
sure!